docs/Guide/getting-started/getting-started-with-sapphire.mdx

```diff
@@ -16,18 +16,40 @@ You need at least Node.js version 16.6.0 to use Sapphire.
 
 :::
 
+:::danger
+
+When using Sapphire it is important that you set a `main` property in your package.json. This is important because we
+use this property to determine where we will need to look for your commands, listeners, events, and other pieces. If you
+don't set this property, we will not be able to find your commands, listeners, events, and other pieces automatically
+for you.
+
+You should set this property to the `"main"` file of your project, this is where you call `client.login()`. For example,
+if your `client.login()` is located at `src/index.js`, then you should set this property to `src/index.js`.
```

Comment:
```suggestion
When using Sapphire, it is important that you set a `main` property in your package.json. This is used to determine 
where we will need to look for your commands, listeners, and other pieces. If you don't set this property, we will 
not be able to automatically find your bot's pieces.

You should set this property to the main file of your project; where you call `client.login()`. For example, if your 
`client.login()` is located at `src/index.js`, then you should set your `main` to `src/index.js`.
```
No need to repeat that it's used to find "commands, listeners, events, and other pieces"; also a few instances where "it" and "this" can be used instead of specifying "this property" every time.